blockchain/chain/src/store/chain_store.rs

```diff
@@ -303,12 +307,12 @@ where
             return Ok(None);
         }
         // TODO get tipset by height using cache instead of reloading tipsets
```

Comment:
Is this comment still relevant? Seems like you implement that caching in tipset_from_keys now

Reply:
ah, nice catch